src/shared/models/GithubUserModel.js

```diff
@@ -0,0 +1,6 @@
+import { UserModel } from './UserModel';
+export class GithubUserModel extends UserModel {
+    static fromOAuth0(idTokenPayload) {
+        return Object.assign(new GithubUserModel(), idTokenPayload);
```

Comment:
not sure, why is this here?

Reply:
@Tuhaj could you please explain more? for example, why do you think it should not be there?

It is just a DTO of user to rely on instead of a pure {} - a user will have a common representation as an specified object.  `UserModel` should have a base methods like `getDisplayName()` and `getEmail()`; if we are going to add different login methods, the DTO / fields names should does not matter (SOLID - Liskov substitution principle)